packages/core/src/components/Notification/Wrappers/NotificationDrawerPanelWrapper.js

```diff
@@ -88,14 +89,14 @@ const NotificationDrawerPanelWrapper = ({
           data-toggle="mark-all-read"
         >
           <Button bsStyle="link" onClick={() => onMarkPanelAsRead(panelkey)}>
-            Mark All Read
+            {translations.readAll}
```

Comment:
@gilad215  just curious what happens when someone accidentally replaces the entire `translations` object instead of merging or `Object.assigning` it, or is that even a concern? If we are could a possible solution involve placing checks for the translation object property, or maybe they're required?

Reply:
`translations` is not a required prop.
I don't see how someone can accidentally override the default object without actually writing `translations={...}` 
If he makes mistakes, thats his responsibility to fix it

Reply:
For clarity, if a consumer only handles these properties ...
```
translations: {
  emptyState: 'No Notifications Available',
  readAll: 'Mark All Read',
  clearAll: 'Clear All'
}
```
What's the expect outcome?

Reply:
The title will be empty, Do you think consumers will want to translate only some of the properties? seems weird to me. Also, having a prop for each component doesn't look like a solution either, this is a very rare case..